aws/resource_aws_fsx_lustre_file_system.go

```diff
@@ -503,9 +497,8 @@ func resourceAwsFsxLustreFileSystemDelete(d *schema.ResourceData, meta interface
 	}
 
 	log.Println("[DEBUG] Waiting for filesystem to delete")
-
-	if err := waitForFsxFileSystemDeletion(conn, d.Id(), d.Timeout(schema.TimeoutDelete)); err != nil {
-		return fmt.Errorf("Error waiting for filesystem (%s) to delete: %w", d.Id(), err)
+	if _, err := waiter.FileSystemDeleted(conn, d.Id(), d.Timeout(schema.TimeoutDelete)); err != nil {
+		return fmt.Errorf("error waiting for FSx lustre filesystem (%s) to deleted: %w", d.Id(), err)
```

Comment:
Consistent capitalization:

```suggestion
		return fmt.Errorf("error waiting for FSx Lustre File System (%s) to deleted: %w", d.Id(), err)
```